.github/workflows/mypy.yaml

```diff
@@ -17,31 +17,7 @@ on:
 jobs:
   test_run:
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.8'
-
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip tox
-        tox -e mypy --notest
-
-    # You can test your matrix by printing the current Python version
-    - name: Display Python information
-      run: |
-        python --version
-        python -c "import sys; print(sys.prefix)"
-        python -c "import sys; print(sys.exec_prefix)"
-        python -c "import sys; print(sys.executable)"
-        python -c "import struct; print(struct.calcsize('P') * 8)"
-        python -c 'print("\nENVIRONMENT VARIABLES\n=====================\n")'
-        python -c "import os; [print(k, v) for k,v in os.environ.items()]"
-
-    - name: Test with mypy
+    - name: Check manifest-checker
       run: |
-        tox -q -e mypy
+        echo
```

Comment:
I could not delete `mypy/test_run`, because it looks like the Twisted GitHub repository is configured to have this as a mandatory workflow.  But we can re-configure the GitHub repo and delete this as necessary.

Reply:
Done. The GitHub UI is a bit confusing here... the check was only listed as "test_run" inside the branch protection part.

Reply:
OK, I deleted mypy.yaml